bitrise.yml

```diff
@@ -404,6 +405,71 @@ workflows:
               exit 1
             fi
 
+  glob_checks:
+    description: |
+      Tests if glob matching works.
+    envs:
+    - CACHE_PATH: |
+        ~/.bitrise/**/*.yml -> ~/.bitrise/plugins/spec.yml
+        ./DoesNotExist/
+        ./DoesNotExist/ -> ./doesnotexist.yml
+        ./DoesNotExist
+        ~/DoesNotExist
+        ~/.bitrise/plugins/step/*/*.go
+    - TMP_DIR: $ORIG_BITRISE_SOURCE_DIR/_tmp_glob
+    steps:
+    - script:
+        title: Cleanup archive tmp files
+        inputs:
+        - content: rm -rf "$TMP_DIR" "/tmp/cache-archive.tar" "/tmp/cache-info.json"
+    - change-workdir:
+        title: Switch working dir to $TMP_DIR
+        inputs:
+        - path: $TMP_DIR
+        - is_create_path: true
+    - path::./:
+        title: Step Test
+        is_always_run: true
+        run_if: true
+        is_skippable: false
+        inputs:
+        - compress_archive: true
+        - is_debug_mode: true
+        - cache_paths: $CACHE_PATH
+    - script:
+        title: Check if stored structure exists
+        is_always_run: true
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+
+            cd $TMP_DIR
+
+            # extract the archive
+            tar -xf /tmp/cache-archive.tar
+
+            if [ ! -f $TMP_DIR/$HOME/.bitrise/plugins/spec.yml ]; then
+              echo "file should exist"
```

Comment:
nit: Printing the exact filename that should exist. All of these checks return the same error string, it would be hard to debug a failing CI build

Reply:
The check is also printed on the `stdout` so I think it is fine this way. (Copied from other tests)

![image](https://user-images.githubusercontent.com/13942888/131848384-58afdae3-e635-4614-80b7-ea726394bd9a.png)